abci/cmd/abci-cli/abci-cli.go

```diff
@@ -54,7 +54,7 @@ var RootCmd = &cobra.Command{
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
 
 		switch cmd.Use {
-		case "kvstore", "version":
+		case "kvstore", "version", "help [command]":
```

Comment:
Thanks for the patch, Jay.
This fixes the issue with the automatically generated help command reported.